src/redux/books/books.js

```diff
@@ -1,30 +1,77 @@
-import bookList from '../../FakeBookData';
+import booksURL from '../../apiURL';
+
 // Action
 const ADD_BOOK = 'ADD_BOOK';
 const DELETE_BOOK = 'DELETE_BOOK';
+const GET_BOOKLIST = 'GET_BOOKLIST';
 
 // Action creators
+// Add book
+export const addBook = (book) => async (dispatch) => {
+  const response = await fetch(booksURL, {
```

Comment:
All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `fetch` OR `axios` ✔

Reply:
Hi @ mateo951

First of all, thanks for taking the time to review the code. I really appreciate it.

About this issue, I believe that in the requirements they give us the option to work with fetch OR axios with AsyncThunk, as seen in the [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md#:~:text=Use%20fetch%20or%20axios%20with%20createAsyncThunk%20for%20making%20API%20calls.)

A copy from the page here:
"Use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) or [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) for making API calls."

After I get approved and If I have time I'll use this to learn it better, but my approval musn't depend on it, since I had another option.

I hope you undersand it.